client/src/components/Type/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './style.css';
+
+const Type = ({ id }) => {
+    let emptyOption;
+    if (id === "type2") {
+        emptyOption = <option value=""></option>;
+    }
```

Comment:
can also make moves2-4 have an empty option since we should be making them optional